src/javascript/JContent/ContentRoute/ContentLayout/Upload/UploadItem/UploadItem.jsx

```diff
@@ -61,7 +59,7 @@ export class UploadItem extends React.Component {
         this.state = {
             userChosenName: null,
             anchorEl: null,
-            numTags: '-'
+            tagComp: ''
```

Comment:
just name component to be more generic, rather set it to null as default to keep type consistent

Reply:
Sounds good updated